cmd/lookout/serve.go

```diff
@@ -151,22 +175,36 @@ func (c *ServeCommand) handlePR(e *lookout.ReviewEvent) error {
 		"head":       e.Head.ReferenceName,
 	})
 	logger.Infof("processing pull request")
-	resp, err := c.analyzer.NotifyReviewEvent(context.TODO(), e)
-	if err != nil {
-		logger.Errorf(err, "analysis failed")
-		return nil
-	}
 
-	if len(resp.Comments) == 0 {
-		logger.Infof("no comments were produced")
-		return nil
+	var comments []*lookout.Comment
+
+	for name, a := range c.analyzers {
+		aLogger := logger.With(log.Fields{
+			"analyzer": name,
+		})
+
+		resp, err := a.NotifyReviewEvent(context.TODO(), e)
```

Comment:
This would block on every analyzer, waiting for response, before sending a request to the next one, wouldn't it?

Reply:
true. Thanks! I was still in a mindset of sdk where everything should be synchronous. 

I update PR using WaitGroup & Mutex.
Though we might use another solution later.